src/Math/Core/PlaneD.cs

```diff
@@ -365,15 +365,7 @@ public override bool Equals(object obj)
         /// </summary>        
         public override int GetHashCode()
         {
-            unchecked
-            {
-                int hash = 17;
-                hash = hash * 29 + A.GetHashCode();
-                hash = hash * 29 + B.GetHashCode();
-                hash = hash * 29 + C.GetHashCode();
-                hash = hash * 29 + D.GetHashCode();
-                return hash;
-            }
+            return HashCode.Combine(A, B, C, D);
```

Comment:
@RedImp1470 `HashCode.Combine()`  is the better method here, isn't it?

Reply:
Yes, this should be the case.